examples/cloud-backup-restore-jobs/restore.json

```diff
@@ -40,7 +45,7 @@
           "Ref": "ProjectId"
         },
         "ClusterName": {
```

Comment:
are we keeping ClusterName?

Reply:
this is for the snapshot, it has not been changed yet